public/app/frontend/style.scss

```diff
@@ -0,0 +1,312 @@
+    font-weight: var(--heading-font-bold);
+    color: var(--justice-primary--black);
+    @include spacing(8,'bottom');
+    .eyebrow {
+        @include eyebrow('h1');
+    }
+    @media screen and (min-width: $justice-lg) {
+        font-size: 3rem;
+        line-height: 3.125rem;
+    }
+}
+
+@mixin h2 {
+    font-size: 1.688rem;
+    line-height: 1.875rem;
+    font-family: var(--heading-font);
+    font-weight: var(--heading-font-bold);
+    color: var(--justice-primary-black);
+    .eyebrow {
+        @include eyebrow('h2');
+    }
+    @media screen and (min-width: $justice-lg) {
+        font-size: 2.25rem;
+        line-height: 2.5rem;
+    }
+}
+
+@mixin h3 {
+    font-size: 1.313rem;
+    line-height: 1.563rem;
+    font-family: var(--heading-font);
+    font-weight: var(--heading-font-bold);
+    color: var(--justice-primary-black);
+    .eyebrow {
+        @include eyebrow('h3');
+    }
+    @media screen and (min-width: $justice-lg) {
+        font-size: 1.5rem;
+        line-height: 1.875rem;
+    }
+}
+
+// TODO: GDS only uses four heading sizes for accessibility reasons - how do we handle this as Justice UK has existing h5/h6 headings?
+@mixin h4 {
+    font-size: 1.188rem;
+    line-height: 1.563rem;
+    font-family: var(--heading-font);
+    font-weight: var(--heading-font-bold);
+    color: var(--justice-primary-black);
+}
+
+@mixin h5 {
+    @include h4;
+    font-weight: var(--heading-font-normal);
+}
+
+@mixin h6 {
+    @include h5;
+}
+
+@mixin body {
+    font-size: 1rem;
+    font-family: var(--body-font);
+    font-weight: var(--body-font-normal);
+    color: var(--justice-primary-black);
+}
+// Heading caption https://design-system.service.gov.uk/styles/headings/#headings-with-captions
+@mixin eyebrow ($tag) {
+    display: block;
+    font-family: var(--heading-font);
+    font-weight: var(--heading-font-normal);
+    color: var(--justice-secondary-grey);
+    @if ($tag == 'h1') {
+        font-size: 1.125rem;
+        line-height: 1.11;
+        @media screen and (min-width: $justice-lg) {
+            font-size: 1.6875rem;
+        }
+    }
+    @if ($tag == 'h2') {
+        font-size: 1.125rem;
+        line-height: 1.11;
+        @media screen and (min-width: $justice-lg) {
+            font-size: 1.5rem;
+            line-height: 1.25;
+        }
+    }
+    @if ($tag == 'h3') {
+        font-size: 1rem;
+        line-height: 1.25;
+        @media screen and (min-width: $justice-lg) {
+            font-size: 1.1875rem;
+            line-height: 1.32;
+        }
+    }
+}
+
+// Icons
+$iconTypes: (
+    'pdf': 'png',
+    'zip': 'gif',
+    'winzip': 'gif'
+);
+
+$iconSizes: (
+    'sm': 1.5rem, // The smallest icon must be at least 24px x 24px to meet WCAG 2.2 AA
+    'md': 3rem,
+    'lg': 6rem
+);
+
+.icon {
+    @each $size, $rem in $iconSizes {
+        @each $icon, $ext in $iconTypes {
+            &-#{$icon}--#{$size} {
+                background-image: url(/src/assets/icons/#{$icon}.#{$ext});
```

Comment:
Is there a method to decide if a component should have global styles here, like .icon ... or be a mixin like `@mixin link` below?

This is something where I've not really known for sure on previous projects.

Reply:
@EarthlingDavey good question.

I think it this case it was purely so that I could use it in the Storybook icon examples page without having to style the page, but that's not a very good reason so I'll change it 😅 

Generally I would prefer most things to be a mixin rather than a global style. Particularly with BEM it means we're namespacing everything.

In this case if we used `.icon` and Wordpress used `.icon` there could be some conflicts, whereas if we have `header__icon` with `@include icon` we're less likely to have that issue.

Reply:
Nice, that makes sense!